src/le.py

```diff
@@ -1235,8 +1238,11 @@ def config_formatters():
 
 def extract_token(log_):
     """Extract the log token value if it exists"""
-    return utils.safe_get(log_, 'log', 'tokens')[0]
-
+    try:
```

Comment:
It doesnt work for ini files or json files. This is the stack trace if changed this way.
  File "/home/mgarewal/rapid7/logentries-json/src/le.py", line 2183, in <module>
    main()
  File "/home/mgarewal/rapid7/logentries-json/src/le.py", line 2175, in main
    main_root()
  File "/home/mgarewal/rapid7/logentries-json/src/le.py", line 2168, in main_root
    return func(args[1:])
  File "/home/mgarewal/rapid7/logentries-json/src/le.py", line 1532, in cmd_monitor
    start_followers(default_transport, state)
  File "/home/mgarewal/rapid7/logentries-json/src/le.py", line 1288, in start_followers
    log_token = extract_token(log_)
  File "/home/mgarewal/rapid7/logentries-json/src/le.py", line 1227, in extract_token
    return utils.safe_get(log_, 'log', 'tokens')[0]
IndexError: list index out of range